### PR TITLE
repo: introduce new Fedora and RHEL versions, remove the ones that are EOL

### PR DESCRIPTION
**Pull request by ondrejbudai**
- Move rawhide to F38
- Introduce F37 branched
- Remove F34
- Remove RHEL 8.6 and 9.0 nightlies
- Introduce RHEL 8.8 and 9.2 nightlies

### CODE REVIEW

File: repo/el8-aarch64-appstream-n8.8.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+        "base-url": "http://download.eng.bos.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.8/compose/AppStream/aarch64/os/",
+        "platform-id": "el8",
+        "snapshot-id": "el8-aarch64-appstream-n8.8",
```
Comment (achilleas-k) [inline repo/el8-aarch64-appstream-n8.8.json:4]: Commit message here is wrong.

**Review by gicmo — DISMISSED** [review]

**Review by achilleas-k — COMMENTED** [review]
Can you fix the commit message on the last one?

File: repo/el8-aarch64-appstream-n8.8.json (hunk 1 above)
Comment (ondrejbudai) [reply repo/el8-aarch64-appstream-n8.8.json:4]: Fixed, thanks!

**Comment by ondrejbudai** [comment]
`-branched` renamed to `-development` and fixed the commit message.

**Review by achilleas-k — APPROVED** [review]

**Pull request merged**